ui/src/organizations/components/OrgTaskExportOverlay.tsx

```diff
@@ -4,51 +4,53 @@ import {withRouter, WithRouterProps} from 'react-router'
 // Components
 import ExportOverlay from 'src/shared/components/ExportOverlay'
 
+// Utils
+import {taskToTemplate} from 'src/shared/utils/resourceToTemplate'
+
 // APIs
 import {client} from 'src/utils/api'
-import {Task} from '@influxdata/influx'
+import {Task} from 'src/types/v2'
 
 interface State {
-  task: Task
+  taskTemplate: Record<string, any>
 }
 
 interface Props extends WithRouterProps {
   params: {id: string; orgID: string}
 }
 
 class OrgTaskExportOverlay extends PureComponent<Props, State> {
-  public state = {task: null}
+  public state = {taskTemplate: null}
+
   public async componentDidMount() {
     const {
       params: {id},
     } = this.props
 
-    const task = await client.tasks.get(id)
-    console.log(task)
-    this.setState({task})
+    const task = (await client.tasks.get(id)) as Task
```

Comment:
Totally not for this PR, but it would be nice to do this through Redux eventually.

Reply:
I know, I'll make an issue for it

Reply:
https://github.com/influxdata/influxdb/issues/12397